### ISSUE TITLE
httpdbg doesn't work with django manage.py

### ISSUE THREAD

**Issue by imankulov** [cle-b/httpdbg#153]
I tried to use httpdbg with my Django project, but I found that it doesn't work well with django manage.py. Can you please help me understand what the problem is?

Steps to reproduce

## Preparatory steps

Create an empty virtual environment, install Django and httpdbg.

```
python3.11 -m venv env
pip install django httpdbg
django-admin startproject foo
cd foo
```

## Run Django server without httpdbg

Works perfectly.

```
$ ./manage.py runserver

Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).

You have 18 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
October 28, 2024 - 19:34:47
Django version 5.1.2, using settings 'foo.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
```

## Run Django server with httpdbg

The script has started and ended immediately

```
$ pyhttpdbg --script ./manage.py runserver
.... - - .--. -.. -... --. .... - - .--. -.. -... --. .... - - .--. -.. -... --.
  httpdbg - HTTP(S) requests available at http://localhost:4909/
.... - - .--. -.. -... --. .... - - .--. -.. -... --. .... - - .--. -.. -... --.
.... - - .--. -.. -... --. .... - - .--. -.. -... --. .... - - .--. -.. -... --.
  httpdbg - HTTP(S) requests available at http://localhost:4909/
.... - - .--. -.. -... --. .... - - .--. -.. -... --. .... - - .--. -.. -... --.
Waiting until all the requests have been loaded in the web interface.
Press Ctrl+C to quit.
$
```

**Comment by cle-b** [comment]
Hello,

There was a bug that prevented to execute the code under `if __name__ == "__main__":` when using the script mode. It fixed now in `httpdbg v0.25.0`. 

I hope this fix is helpful to you.

**Comment by imankulov** [comment]
Awesome! It's working now. Thanks a lot for the quick turnaround.
